new_jersey/ver2/style.css

```diff
@@ -0,0 +1,171 @@
+    src: url('fonts/SourceSansPro-Semibold.eot');
+    src: local('☺'), url('fonts/SourceSansPro-Semibold.woff') format('woff'), url('fonts/SourceSansPro-Semibold.ttf') format('truetype'), url('fonts/SourceSansPro-Semibold.svg') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+@font-face {
+    font-family: 'Proxima Nova Bold';
+    src: url('fonts/Mark Simonson - Proxima Nova Bold.eot');
+    src: local('☺'), url('fonts/Mark Simonson - Proxima Nova Bold.woff') format('woff'), url('fonts/Mark Simonson - Proxima Nova Bold.ttf') format('truetype'), url('fonts/Mark Simonson - Proxima Nova Bold.svg') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+.container {
+    background-color: #4e5860;
+    width: 800px;
+    height: 600px;
+    margin: 0 auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.widget {
+    background-color: #f1f8f9;
+    width: 388.99px;
+    height: 464px;
+    border-radius: 10px 10px 29px 29px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    box-shadow: 0px 37px 76px 2px rgba(0, 0, 0, 0.2);
+}
+.header {
+    display: flex;
+    justify-content: flex-end;
+    padding-top: 16px;
+    padding-right: 17px;
+    border-radius: 10px 10px 0 0;
+}
+.main {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-left: 5px;
+    padding-top: 10px;
+}
+
+.forecast {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 30px;
+}
+
+.info {
+    display: flex;
+    align-items: flex-start;
+    font-family: "Proxima Nova Light";
+    font-size: 58px;
+    margin-left: 20px;
+}
+.temperature {
+    color: #a5acaf;
+}
+.info img {
+    padding-left: 5px;
+    padding-top: 16px;
+}
+
+.description_location {
+    display: flex;
+    flex-direction: column;
+    margin-right: 37px;
+}
+.description {
+    font-family: "SourceSansPro-Light";
+    color: #bac2c6;
+    font-size: 24px;
+}
+.location {
+    font-family: "SourceSansPro-Semibold";
+    color: #a2a9ad;
+    font-size: 16px;
+}
+
+.date {
+    background-color: #54bae6;
+    width: 65px;
+    height: 69px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    font-family: "Open Sans", Sans-Serif;
+    font-weight: 700;
+    color: #ffffff;
+}
+.month {
+    padding-top: 5px;
+    text-transform: uppercase;
+}
+
+.footer {
+    background-color: #2e3335;
+    width: 388.99px;
+    height: 136px;
+    border-radius: 0px 0px 10px 10px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.additional-info {
+    width: 388.99px;
+    margin-top: 46px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    font-family: "Proxima Nova Bold";
+    color: #ffffff;
+    font-size: 14px;
+    text-transform: uppercase;
+}
+.wind {
```

Comment:
the "wind", "rain" and "sun" sections are essentially the same, so, in order to avoid the repetitions in our code, I would suggest changing our layout to:
```
<div class="additional-info">
   <div class="additional-info__item">
   <div class="additional-info__item">
   <div class="additional-info__item">
</div>
```
Next, we can change the "additional-info__item" style to something like:
```
    width: 30%;
    display: flex;
    align-items: center;
    justify-content: center;
```
This will eliminate the need for creating specific left & right margins in px and will allow us to reuse our code.

Reply:
Actually it made code more readable and comfortable to fix in further, but anyway I had to apply to individual elements `.additional-info__item` via pseudo-classes `:nth-child` and `:last-child` to align them perfectly :octocat:

Reply:
Using pseudo-classes to apply styles unique to the first & the last element instead of creating classes is considered best practice. Good job!

Reply:
In fact, before your review i thought, that using pseudo-classes is like a crutch in css 😄 Now I will use it further without doubt. Thank you!